deployer/templates/fluentd.yaml

```diff
@@ -329,13 +323,9 @@ parameters:
   name: USE_MUX
   value: "false"
 -
-  description: 'Configure MUX CLIENT (false|true).'
-  name: USE_MUX_CLIENT
-  value: "false"
--
-  description: 'Configure MUX SERVER (false|true).'
-  name: MUX_ALLOW_EXTERNAL
-  value: "false"
+  description: 'Configure MUX CLIENT MODE (minimal|full_no_k8s_meta).'
```

Comment:
Can we use a description like, "Configure fluentd collector MUX CLIENT MODE ..."